src/PetProjects.Transactions.Contracts/Topics/V1/TransactionEventsTopic.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace PetProjects.Transactions.Contracts.Topics.V1
+{
+    using Events;
+    using Framework.Kafka.Contracts.Topics;
+
+    public class TransactionEventsTopic : ITopic<TransactionEvents>
+    {
+        public string TopicFullName => this.SetTopicName().TopicFullName;
+
+        public TopicBuilder SetTopicName()
+        {
+            return new TopicBuilder("transaction", MessageType.Events)
+                .WithApplication("transactions-api");
```

Comment:
Maybe "mts" or "micro-transactions-system" instead of "transactions-api" ?

Reply:
The idea for the Application was to make it visible to who application the topic belonged. But since it is the whole system, mts is a good name.

Reply:
Hmm actually it might be a good idea to make it "transactions-api" after all if we consider that these messages are owned by the transactions api.

Reply:
done